src/styles/index.scss

```diff
@@ -1,3 +1,141 @@
+$body-font: "Roboto", sans-serif;
+$font-size: 10px;
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-blue-accent: #00acdc;
+
 body {
   margin: 0;
+
```

Comment:
Why empty space? There are a lot of empty lines in weird places. You can remove them :)